frontend21/scss/amp-dev.scss

```diff
@@ -1,5 +1,55 @@
-@import '../frontend/scss/_functions.scss';
-@import '../frontend/scss/_mixins.scss';
-@import '../frontend/scss/_variables.scss';
+@import '../../frontend/scss/_functions.scss';
+@import '../../frontend/scss/_mixins.scss';
+@import '../../frontend/scss/_variables.scss';
+@import '../../frontend/scss/components/atoms/_text.scss';
+@import '../../frontend/scss/components/atoms/_color.scss';
 
+@import 'components/section.scss';
 @import 'components/text-media.scss';
+@import 'components/text-block.scss';
+@import 'components/button.scss';
+@import 'components/pie-chart.scss';
+
+main {
+  padding: 0;
+  width: 100%;
+  max-width: 1920px;
+  margin: auto;
+
+  @media (min-width: 768px) {
+    display: grid;
+    grid-gap: 30px;
+    grid-template-columns: repeat(24, minmax(0, 1fr));
```

Comment:
24 cols seems a little granular. Doesn't 12 already work or does that cut on Ferdi's design freedom? What use-case does the `minmax` serve here?